### ISSUE TITLE
OpenAsar Settings Location?

### ISSUE THREAD

**Issue by MonfGeiger** [GooseMod/OpenAsar#181]
Whenever my discord updates, it removes openasar, and i have to reinstall it. then whatever css theme i was using is reapplied. Is there some folder location openasar uses for their settings/css stuff......or is it all a registry thing?

**Comment by k2helix** [comment]
It is the settings.json file discord uses too.

**Comment by lilmayu** [comment]
As the other person said, but without actual path. It is in the `C:\Users\<USER>\AppData\Roaming\discord` directory in the `settings.json` file.

**Comment by k2helix** [comment]
Yup, I didn't actually specify the path as I use it on Linux and don't know the folder used on Windows. For me it is `~/.config/Discord/settings.json`
